broadbean/blueprint.py

```diff
@@ -328,7 +342,7 @@ def init_from_json(cls, path_to_file: str) -> 'BluePrint':
             data_loaded = json.load(fp)
         return cls.blueprint_from_description(data_loaded)
 
-    def _makeWaitDurations(self):
+    def _make_wait_durations(self) -> Sequence[int]:
```

Comment:
it's ok to say what's returned exactly (as opposed to input arguments where the it's better to declare as few restrictions in the type as possible because then the functions become more applicable), and in this case a list is returned (right?).
```suggestion
    def _make_wait_durations(self) -> List[int]:
```

Reply:
I see what you mean thanks for the clarification!